From 84ab511a62baeb183320e72e1271cf255581478c Mon Sep 17 00:00:00 2001
From: Philippe Bellanger <pbellanger@hotmail.com>
Date: Wed, 7 Feb 2018 21:03:29 -0500
Subject: [PATCH] Added Chrome overrides to skip Webview and WebViewGoogle

---
 modules/Chrome/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/Chrome/Android.mk b/modules/Chrome/Android.mk
index cf46b33..cc19faf 100644
--- a/modules/Chrome/Android.mk
+++ b/modules/Chrome/Android.mk
@@ -5,6 +5,6 @@ LOCAL_MODULE := Chrome
 LOCAL_PACKAGE_NAME := com.android.chrome
 
 GAPPS_LOCAL_OVERRIDES_MIN_VARIANT := stock
-GAPPS_LOCAL_OVERRIDES_PACKAGES := Browser Browser2 BrowserProviderProxy Chromium Fluxion Gello Jelly
+GAPPS_LOCAL_OVERRIDES_PACKAGES := Browser Browser2 BrowserProviderProxy Chromium Fluxion Gello Jelly WebView WebViewGoogle
 
 include $(BUILD_GAPPS_PREBUILT_APK)
-- 
1.9.1

